src/ProtectorServiceProvider.php

```diff
@@ -31,7 +31,7 @@ public function boot(): void
         // Publish package config to app config space.
         $this->publishes([
             __DIR__ . '/../config/protector.php' => config_path('protector.php'),
-        ], 'protector.config');
+        ], ['protector', 'protector.config']);
```

Comment:
does this add a subfolder?

Reply:
No, this is a tag you can reference when using `php artisan vendor:publish`.